aviary/examples/run_new_detailed_takeoff.py

```diff
@@ -0,0 +1,320 @@
+import aviary.api as av
+
+
+throttle_max = 1.0
+throttle_climb = 0.956
+throttle_cruise = 0.8
+throttle_idle = 0.0
+
+
+subsystem_options = {'core_aerodynamics':
+                     {'method': 'low_speed',
+                      'ground_altitude': 0.,  # units='ft'
+                      'angles_of_attack': [
+                          0.0, 1.0, 2.0, 3.0, 4.0, 5.0,
+                          6.0, 7.0, 8.0, 9.0, 10.0, 11.0,
+                          12.0, 13.0, 14.0, 15.0],  # units='deg'
+                      'lift_coefficients': [
+                          0.5178, 0.6, 0.75, 0.85, 0.95, 1.05,
+                          1.15, 1.25, 1.35, 1.5, 1.6, 1.7,
+                          1.8, 1.85, 1.9, 1.95],
+                      'drag_coefficients': [
+                          0.0674, 0.065, 0.065, 0.07, 0.072, 0.076,
+                          0.084, 0.09, 0.10, 0.11, 0.12, 0.13,
+                          0.15, 0.16, 0.18, 0.20],
+                      'lift_coefficient_factor': 1.,
+                      'drag_coefficient_factor': 1.}}
+
+optimize_mach = False
+optimize_altitude = False
+
+# Currently the more complete takeoff trajectory requires SNOPT to converge.
+# If False, the takeoff trajectory will be simplified to a ground roll only
```

Comment:
IPOPT really likes to have reasonably tight upper and lower bounds, so that might help with it s convergence.

Reply:
Thanks Ken, I've added some upper bounds and it allowed me to have a few more phases in the non-SNOPT case. Definitely more work to do but I feel better about it now and with the changes on my other branch than I did last week.